### PR TITLE
add typeplate & organise the mess :rocket:

### LINKED ISSUE #5: Create categories for 'Libraries and Mixins'
Right now this list is a bit of a chaos :P

We could try to split those links into categories.

Some examples:

* Color management
* Typography
* Media queries
* Etc.

### PR DESCRIPTION
**Pull request by flexbox**
Proposition to close #5

### CODE REVIEW

**Comment by tzi** [comment]
Please merge this! :clap:

File: README.md [hunk 1]
```diff
@@ -12,85 +12,101 @@
 - [Sass vs SCSS](#sass-vs-scss)
 - [Frameworks](#frameworks)
 - [Libraries and Mixins](#libraries-and-mixins)
+  - [Animation](#animation)
+  - [Color](#color)
   - [Grid](#grid)
+  - [Media Queries](#mediaqueries)
```
Comment (tandavas) [inline README.md]: Can you please fix this one to be:

```
- [Media Queries](#media-queries)
```
instead?

The link won't work if there's no `-` in place of the space between words (Try clicking it on your repo https://github.com/flexbox/awesome-scss to see).

See more info here: https://gist.github.com/asabaylus/3071099

File: README.md [hunk 2]
```diff
@@ -12,85 +12,101 @@
...
+  - [Misc](#Misc)
```
Comment (tandavas) [inline README.md]: Can fix this line to be `[Misc](#misc)` instead? 

It's supposed to be small `m` after `#`, or else the anchor link won't work 😢

**Comment by tandavas** [comment]
Can you also please keep the order of the list as original, not to be sorted alphabetically? 

Since this list was inspired by https://github.com/sindresorhus/awesome, and I will be eventually send a pull request to add awesome-scss to it. So, I would like to keep things the way @sindresorhus does it (ref https://github.com/sindresorhus/awesome/issues/548#issuecomment-189576619).

By the way, thank you for categorizing this! This is so awesome 😃 🎉 !

**Comment by flexbox** [comment]
Hello there ! 
Sorry for my sorting hysteria :) 
@tandavas maybe you can give me some rights on the repo to helping you maintaint it 
-_- need to sync my PR across forks & I am lazy xD

**Comment by tandavas** [comment]
@flexbox Thank you for helping sorting the list and for offering to help maintain this, but I think we got this. 

I'm closing this and #5 with 0aef2b0410da3b32b27fcf5a859d701a3a00b715  with the correct sorting and working links.

**Pull request closed without merging**